SalesforceCommerceCloud/pwa-kit
# fix translation in extended app

# Description
Currently on extended app, Template Extensibility projects are not able to use translation on overriden files . This PR attempted to fix it by adjusting translation scripts to extract messages from both extended app and base template, so the extended app will have full translations.





# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- template-retail-app/app/utils/locale.js fetchTranslation, we need to adjust the dynamic import from relative import to absolute import so it will get the correct translation files.
- package.json in extended app

# How to Test-Drive This PR

- npm ci
- npm start in my-extended-app
- Add a new translation in the extended app
- Run `npm run build-translations` to extract compile all translations
- Confirm that new added translations are present in the en-US.json, and its compiled file

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
